Frontend/src/test/pages/dashboard/components/Customer/SavedAddresses.test.jsx

```
@@ -0,0 +1,120 @@
+          onClick={() =>
+            onAdd({
+              id: 3,
+              name: 'New Name',
+              street: '789 Pine Rd',
+              city: 'Newcity',
+              state: 'NC',
+              zip: '55555',
+              country: 'USA',
+            })
+          }
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)